script/application.go

```diff
@@ -0,0 +1,227 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"strings"
+	"time"
+
+	"github.com/google/go-github/v60/github"
+)
+
+type Project struct {
+	Name         string `json:"name"`
+	Description  string `json:"description"`
+	Contributors int    `json:"contributors"`
+	HomeURL      string `json:"home_url"`
+	RepoURL      string `json:"repo_url,omitempty"`
+	LicenseType  string `json:"license_type,omitempty"`
+	LicenseURL   string `json:"license_url,omitempty"`
+	IsEvent      bool   `json:"is_event"`
+	IsTeam       bool   `json:"is_team"`
+}
+
+type Applicant struct {
+	Name  string `json:"name"`
+	Email string `json:"email"`
+	Role  string `json:"role"`
+	Id    int64  `json:"id"`
```

Comment:
Minor nit - Technically ID is also an acronym, and convention is to write this as all caps as well :P Same with ApproverID below.

Reply:
🤦 good callout!